app/views/comments/_form.html.haml

```diff
@@ -1,23 +1,22 @@
-= render :partial => 'comments/help'
-
+= render :partial => 'comments/help', locals: { unique_id: unique_id}
 = form_for [ parent, comment ], html: {role: "form"} do |f|
   %p
     .comment-form-heading
       %ul.nav.nav-tabs
-        %li.active#show-source{ role: :presentation }
-          %a{ href: '#markdown-source', role: :tab, data: { toggle: :tab } } Edit
-        %li#show-preview{ role: :presentation }
-          %a{ href: '#markdown-preview', role: :tab, data: { toggle: :tab } } Preview
+        %li.active{ id: "show-source#{unique_id}", role: :presentation }
```

Comment:
This could be `id` instead of `unique_id`, id should be unique. :smile_cat:

Reply:
I tried to keep it different from the word 'id', as I thought it might be confusing for other devs